lib/defines.mk

```diff
@@ -1,4 +1,4 @@
-CFLAGS ?= -O2 -g
+CFLAGS ?= -O2 -g -Werror
```

Comment:
Turns out this did actually break things; could you please add this as well?

`BPF_CFLAGS ?= -Wno-visibility`

Reply:
Odd, did not break anything on my side (compilers :(
Will add it